### PR TITLE
docs(1.7.0): add PSA for issue 9267

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue #

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.7.0.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ The Longhorn team appreciates your contributions and anticipates receiving feedb
 > **Note:**
 > For more information about release-related terminology, see [Releases](https://github.com/longhorn/longhorn#releases).
 
+## PSA
```
Comment (innobead) [inline CHANGELOG/CHANGELOG-1.7.0.md:10]: This is not required for note.

File: CHANGELOG/CHANGELOG-1.7.0.md [hunk 2]
```diff
@@ -7,6 +7,18 @@ The Longhorn team appreciates your contributions and anticipates receiving feedb
...
+
+### Warning
```
Comment (innobead) [inline CHANGELOG/CHANGELOG-1.7.0.md:12]: Use GitHub warning markdown syntax  instead.

**Review by innobead — CHANGES_REQUESTED** [review]

**Comment by innobead** [comment]
Add this warning to the important note section.

File: CHANGELOG/CHANGELOG-1.7.0.md (hunk 1 above)
Comment (derekbit) [reply CHANGELOG/CHANGELOG-1.7.0.md]: Fixed. Thank you.

File: CHANGELOG/CHANGELOG-1.7.0.md (hunk 2 above)
Comment (derekbit) [reply CHANGELOG/CHANGELOG-1.7.0.md]: Updated. Thank you.

File: CHANGELOG/CHANGELOG-1.7.0.md [hunk 3]
```diff
@@ -53,8 +64,7 @@ The storage network now [supports RWX volumes]((https://github.com/longhorn/long
 
 The [Longhorn CLI](https://github.com/longhorn/longhorn/issues/7927), which is the official Longhorn command line tool, is introduced in v1.7.0. This tool interacts with Longhorn by creating Kubernetes custom resources (CRs) and executing commands inside a dedicated pod for in-cluster and host operations. Usage scenarios include installation, operations such as exporting replicas, and troubleshooting.
 
-## Installation
-
+>  [!IMPORTANT]
```
Comment (innobead) [inline CHANGELOG/CHANGELOG-1.7.0.md:67]: Add `!important` to the upgrade section below as well. 

> Ensure that your cluster is running Kubernetes v1.21 or later before upgrading from Longhorn v1.6.x to v1.7.0.

Comment (derekbit) [reply CHANGELOG/CHANGELOG-1.7.0.md:67]: Added. Thank you.

File: CHANGELOG/CHANGELOG-1.7.0.md [hunk 4]
```diff
@@ -4,8 +4,19 @@ This latest version of Longhorn introduces several features, enhancements, and b
 
 The Longhorn team appreciates your contributions and anticipates receiving feedback regarding this release.
 
-> **Note:**
-> For more information about release-related terminology, see [Releases](https://github.com/longhorn/longhorn#releases).
+For more information about release-related terminology, see [Releases](https://github.com/longhorn/longhorn#releases).
+
+
+> [!WARNING]
+> **PSA**
```
Comment (innobead) [inline CHANGELOG/CHANGELOG-1.7.0.md:11]: Remove this PSA word.

**Review by innobead — CHANGES_REQUESTED** [review]

File: CHANGELOG/CHANGELOG-1.7.0.md (hunk 4 above)
Comment (derekbit) [reply CHANGELOG/CHANGELOG-1.7.0.md]: Removed. Thanks.

File: CHANGELOG/CHANGELOG-1.7.0.md [hunk 5]
```diff
@@ -53,14 +62,14 @@ The storage network now [supports RWX volumes]((https://github.com/longhorn/long
 
 The [Longhorn CLI](https://github.com/longhorn/longhorn/issues/7927), which is the official Longhorn command line tool, is introduced in v1.7.0. This tool interacts with Longhorn by creating Kubernetes custom resources (CRs) and executing commands inside a dedicated pod for in-cluster and host operations. Usage scenarios include installation, operations such as exporting replicas, and troubleshooting.
 
-## Installation
```
Comment (innobead) [inline CHANGELOG/CHANGELOG-1.7.0.md:56]: The section needs to be kept.

Comment (derekbit) [reply CHANGELOG/CHANGELOG-1.7.0.md:56]: Fixed.

File: CHANGELOG/CHANGELOG-1.7.0.md [hunk 6]
```diff
@@ -4,8 +4,17 @@ This latest version of Longhorn introduces several features, enhancements, and b
 
 The Longhorn team appreciates your contributions and anticipates receiving feedback regarding this release.
 
-> **Note:**
-> For more information about release-related terminology, see [Releases](https://github.com/longhorn/longhorn#releases).
+For more information about release-related terminology, see [Releases](https://github.com/longhorn/longhorn#releases).
+
+
+> [!WARNING]
+> The Longhorn team has identified [a critical issue](https://github.com/longhorn/longhorn/issues/9267) that affects volume attachment in Longhorn v1.7.0. We are currently working on releasing v1.7.1, which will be available soon. If your Longhorn cluster contains `engine` resources with names in the format `<volume name>-e-<8-char random id>` that are created before v1.5.2 and v1.4.4, please put v1.7.0 upgrade on hold until v1.7.1 is available.
```
Comment (jhkrug) [inline CHANGELOG/CHANGELOG-1.7.0.md:11]: I don't quite get why it's 'before v1.5.2 and v1.4.4`. Is not before v1.5.2 sufficient? Apart from that, all ok.

**Review by jhkrug — APPROVED** [review]
lgtm modulo my minor query

File: CHANGELOG/CHANGELOG-1.7.0.md (hunk 6 above)
Comment (derekbit) [reply CHANGELOG/CHANGELOG-1.7.0.md:11]: Volume created `before v1.4.4`, `v1.5.0` and `v1.5.1` can hit the issue.

**Pull request merged**